### ISSUE TITLE
Predicted Labels seem incorrect

### ISSUE THREAD

**Issue by abfleishman** [olgaliak/active-learning-detect#22]
We have multiple labels in our dataset and it appears as if there is a bug that is assigning all the tags to a single class. We get predictions that accurately put bounding boxes around all of the classes we have trained on but they are labeled as the 'petrel' class  We only get predictions for a two classes (petrel and rat) if you look in the tagged_preds.csv but there are only 3 rows with this classes predicted compared to 3,047 petrel predictions.  Most (~50%??) of the petrel predictions are other classes. 

There is a class imbalance in the training data:
petrel:

class | Count
-- | --
cat | 121
finch | 60
horse | 25
human | 104
kitten | 2
NULL | 3124
petrel | 781
rail | 2
rat | 208
warbler | 1

Let me know what info is needed to troubleshoot/fix this issue?

**Comment by abfleishman** [comment]
I retrained after balancing the classes manually (by removing entries from the csv) and I began getting predictions for other classes.

**Comment by yashpande** [comment]
Good to hear! I'm not sure what pipeline.config file you're using but it might also be helpful to use data augmentation to mitigate the extent that the class imbalance in your training set affects your model. https://stackoverflow.com/questions/44906317/what-are-possible-values-for-data-augmentation-options-in-the-tensorflow-object has a list of options you can use. Let me know if you have any questions about using this!
